tutor/src/models/exercises.js

```diff
@@ -13,7 +13,7 @@ export { COMPLETE, PENDING };
 
 export const exerciseSort = (exercises, course, user) => {
   return sortBy(exercises, (ex) => {
-    const cs = ex.page.chapter_section.asString;
+    const cs = ex.page.chapter_section.asNumber;
     if (ex.belongsToUser(user)) {
```

Comment:
need to use the numeric version of c/s so it sorts correctly otherwise stuff like intro will appear at the end